### PR TITLE
[baxtereus] override :angle-vector and rename existing methods

### PR DESCRIPTION
**Pull request by knorth55**
MERGE AFTER #719 
Before                       After
`:angle-vector`             -> `:angle-vector-raw`
`:angle-vector-sequence`    -> `:angle-vector-sequence-raw`
`:angle-vector-motion-plan` -> `:angle-vector`

related to https://github.com/jsk-ros-pkg/jsk_robot/pull/716#issuecomment-259861725 and https://github.com/jsk-ros-pkg/jsk_robot/pull/716#issuecomment-259869688

it works with simulation and real robot

### CODE REVIEW

**Comment by knorth55** [comment]
@pazeshun can you review this?

**Comment by pazeshun** [comment]
If `mvit-rb` and `mvit-env` are nil, what happens?
I think it's better that we can use `:angle-vector` as `:angle-vetor-raw` in that situation.
@knorth55 What do you think?

**Comment by knorth55** [comment]
> If mvit-rb and mvit-env are nil, what happens?

baxter-interface do not initialize moveit config.

https://github.com/knorth55/jsk_robot/blob/b8464bc2345a19a55e75c04dfb56b31c72286024/jsk_baxter_robot/baxtereus/baxter-interface.l#L49-L50

> I think it's better that we can use :angle-vector as :angle-vetor-raw in that situation.

method `:angle-vector` should be only for moveit, and in other case, robot should not move because it can be dangerous

**Comment by pazeshun** [comment]
Ok, I understood.
+1

**Comment by pazeshun** [comment]
https://github.com/jsk-ros-pkg/jsk_robot/pull/716#issuecomment-260166997

**Comment by knorth55** [comment]
@pazeshun modified

**Comment by knorth55** [comment]
modified

**Comment by pazeshun** [comment]
+1

File: jsk_baxter_robot/baxtereus/baxter-interface.l [hunk 1]
```diff
@@ -232,14 +232,32 @@
...
+- av : joint angle vector [rad]
+- tm : (time to goal in [msec]) ;; currently this value is ignored
+"
+   ;; for simulation mode
+   (when (send self :simulation-modep)
+     (return-from :angle-vector (send* self :angle-vector-raw av tm args)))
+   (when (not (numberp tm))
+     (ros::warn ":angle-vector tm is not a number, use :angle-vector av tm args"))
+   (if (and (get self :moveit-environment)
+            ((get self :moveit-environment) . robot))
```
Comment (knorth55) [inline jsk_baxter_robot/baxtereus/baxter-interface.l]: thank you!

**Comment by knorth55** [comment]
fixed

**Pull request merged**